lib/metrics/amplitude.js

```diff
@@ -105,6 +105,11 @@ module.exports = (log, config) => {
     throw new TypeError('Missing argument');
   }
 
+  const verificationReminders = require('../verification-reminders')(log, config);
+  verificationReminders.keys.forEach(key => {
```

Comment:
~Since we have a config defining the types of reminders, does it make sense to pull it directly into `EMAIL_TYPES` above?~

After going through the code a bit more, I think this is ok. The config file *almost* defines everything needed for the emails except the template contents. The amount of work/overhead required for that is probably not worth it for this version.

Reply:
Yeah, I was trying to see how far I could get towards it being completely data-driven, and just the templates are the blocker to that. But there's still benefit to it without those, because at least it means there'll be less places to edit if we e.g. reduce it to 1 reminder or increase it to 3 (or whatever).